js/components/tooltip.js

```diff
@@ -4,6 +4,7 @@ var deepmerge = require('deepmerge');
 var Utils = require('./utils');
 var createReactClass = require('create-react-class');
 var PropTypes = require('prop-types');
+var CONSTANTS = require('../constants/constants');
```

Comment:
Do you want to update these to 'const' as well?

Reply:
I just updated the new files. We'll have to migrate the older ones gradually, but it's ok to keep them like this for now.